### PR TITLE
Apply new styling

### PR DESCRIPTION
**Pull request by inglesp**
@hawkz will provide the raw SASS files in due course.

### CODE REVIEW

**Comment by inglesp** [comment]
Could somebody (@ghickman?) review this to double check that this isn't going to muck up the holding page at http://pyconuk.org?

File: content/code-of-conduct-handling-reports.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 type: page
 url: code-of-conduct/handling-reports/index.html
+title: Handling reports of harrassment
```
Comment (d0ugal) [inline content/code-of-conduct-handling-reports.md:3]: harrassment -> harassment

File: content/code-of-conduct-reporting-harrassment.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 type: page
 url: code-of-conduct/reporting-harrassment/index.html
+title: Reporting harrassment
```
Comment (d0ugal) [inline content/code-of-conduct-reporting-harrassment.md:3]: harrassment -> harassment

File: content/code-of-conduct-reporting-harrassment.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
 ---
 
 ## How to report harrassment
```
Comment (d0ugal) [inline content/code-of-conduct-reporting-harrassment.md:6]: harrassment -> harassment :smile: 

Might be worth a search and replace for this.

**Comment by ghickman** [comment]
@inglesp – I ran your branch locally and the holding page looks fine to me 👍

**Comment by snim2** [comment]
This looks good to me, but the page titles look liket hey have a font missing:

snim2-home

**Comment by meshy** [comment]
:+1: From me on Firefox for Mac

**Comment by inglesp** [comment]
I don't like merging things myself, but I want to get on with this, so I'm going to.  (And there've been several "looks goods".)

Have opened https://github.com/PyconUK/2016.pyconuk.org/issues/20 to track the fonts problem

**Comment by hawkz** [comment]
The font will work on 127.0.0.1 and the live domain. It's a licensing thing. I'll try to add 0.0.0.0

~Steve

**Pull request merged**